interface/forms/observation/new.php

```diff
@@ -23,7 +23,7 @@
 use OpenEMR\Core\Header;
 
 $returnurl = 'encounter_top.php';
-$formid = 0 + (isset($_GET['id']) ? $_GET['id'] : 0);
+$formid = 0 + (isset($_GET['id']) ? $_GET['id'] : '');
```

Comment:
Will not bring in this specific change into codebase.

Reply:
I am a bit confused. Was it not already there?

Reply:
Jerry set this value to 0 (here and at a couple places) in the main codebase a couple days ago in this commit:
https://github.com/openemr/openemr/commit/353169ddb5b03095d4d4c54bb8917b2aa24a9946

And then your code basically reverts this. No biggie, I am just posting these messages so I don't forget.